dev-python/pyotp/pyotp-2.2.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 )
+inherit distutils-r1
+
+DESCRIPTION="dev-phyton/pyotp 2.2.1"
+HOMEPAGE="https://github.com/pyotp/pyotp/"
+SRC_URI="https://github.com/pyotp/pyotp/archive/v2.2.1.tar.gz"
```

Comment:
Please use `mirror://pypi/...` as the `SRC_URI`  and use `${PN}` `${P}` for package name and version where appropriate. See also [Variables – Gentoo Development Guide](https://devmanual.gentoo.org/ebuild-writing/variables/index.html)